services/basic_types.proto

```diff
@@ -1385,9 +1385,9 @@ message ServiceEndpoint {
 /**
  * The data about a node, including its service endpoints and the Hedera account to be paid for
  * services provided by the node (that is, queries answered and transactions submitted.)
- * 
- * If the <tt>serviceEndpoint</tt> list is not set, or empty, then the endpoint given by the
- * (deprecated) <tt>ipAddress</tt> and <tt>portno</tt> fields should be used.
+ *
+ * If the `serviceEndpoint` list is not set, or empty, then the endpoint given by the
```

Comment:
I believe so, as this is a field name in the same message, so it's a code reference.
The guideline I'm following is to use inline code styling (\`) for references to field or message names.

Reply:
I find the mixture of html and markdown for formatting confusing. But if this is the standard we want to follow...

Reply:
It's all markdown; it's just that base markdown has two large gaps normally filled with html tags.
1. Standard markdown has inconsistent support for a line break.  It adds a paragraph break for each blank line, but single line break is recommended to use `<br/>` for best compatibility.
1. Markdown requires the use of HTML tags for anything multi-line in tables because tables are an afterthought in markdown so a table row cannot contain a newline.
What the processor does is remove all line breaks in these comment blocks before building the table row, so we have to use tags a bit more in field comments than in message comments.
The main alternative would be to use html exclusively, but that's much harder to read, in my opinion, than markdown.